ATM.py

```diff
@@ -0,0 +1,106 @@
+"""
+
+The script simulates ATM that preforms the following:
+balance check, withdraw cash, deposit cash and
+password change.
+
+"""
+import os
+import sys
+
+
+PATH=1
+BALANCE=1
+PASSWORD=0
+
+
+def file_to_dic(filename):
+    """
+
+    :arg
+        filename(.txt file) - parameter
+
+    :return:
+        a dictionary, key=id and value=[password, balance]
+
+    """
+    atm = {}
+    with open(filename, 'r') as file:
+        for line in file:
+            details = line.strip().split(' ')
+            atm[details[0]] = details[1:]
+
+    return atm
+
+
+def atm_machine(id, atm_dic):
+    """
+
+    Args:
+        id (str)- first parameter
+        atm_dic (dic)- second parameter
+
+    :return:
+        A dictionary with the updated values
+        after performing the desired action
+
+    """
+    if atm_dic.get(id):
+        action = input("enter number of action:\n"
+                       "1- for check the balance\n"
+                       "2- for cash withdrawal\n"
+                       "3- for cash deposit\n"
+                       "4- for change password\n")
+
+        if action == '1':
+            print(f"your balance is: {atm_dic[id][BALANCE]}")
+
+        elif action == '2':
+            money = input("enter the amount you want to withdraw\n")
+            atm_dic[id][BALANCE] = str(int(atm_dic[id][BALANCE]) - int(money))
+
+        elif action == '3':
+            money = input("enter the amount you want to deposit\n")
+            atm_dic[id][BALANCE] = str(int(atm_dic[id][BALANCE]) + int(money))
+
+        elif action == '4':
+            password = input('enter new password\n')
+            atm_dic[id][PASSWORD] = password
+
+        else:
+            print('action not detected')
+
+    else:
+        print('there is an error i the ID you etered')
```

Comment:
when you print something to the terminal (like logs or system messages) always start the message sentence with a capital letter.

for example:
```suggestion
        print('There is an error i the ID you etered')
```